VascularFlow/Numerics/Assembly.py

```diff
@@ -18,3 +17,47 @@ def assemble_system_matrix(element_matrices_enn: ndarray):
         ] += element_matrices_enn[e]
 
     return system_matrix_gg
+
+
+def assemble_system_matrix_2dof(element_matrices_enn: ndarray):
+    """Assemble the system matrix from the element matrices."""
+    nb_elements, nb_nodes, _ = element_matrices_enn.shape
+    assert element_matrices_enn.shape == (nb_elements, nb_nodes, nb_nodes)
+    nb_global_nodes = nb_elements * 2 + 2
+
+    system_matrix_gg = np.zeros((nb_global_nodes, nb_global_nodes))
+
+    for e in range(nb_elements):
+        start = e * (nb_nodes - 2)
+        end = start + nb_nodes
+        system_matrix_gg[
+            start : end,
+            start : end,
+        ] += element_matrices_enn[e]
+    return system_matrix_gg
+
+def assemble_force_matrix_2dof(element_matrices_enn: ndarray):
+    """Assemble the system matrix from the element matrices."""
+    nb_elements, _, nb_nodes = element_matrices_enn.shape
+    nb_global_nodes = nb_elements * 2 + 2
+
+    system_matrix_gg = np.zeros(nb_global_nodes)
+
+    for e in range(nb_elements):
+        start = e * (nb_nodes - 2)
+        end = start + nb_nodes
+        system_matrix_gg[start:end] += element_matrices_enn[e].reshape(4,)
+    return system_matrix_gg
+
+
+def assemble_force_matrix_pressure(element_matrices_enn: ndarray):
```

Comment:
Is this the force vector? (Then please rename function to assemble the vector.) How is this specific to pressure? Please also update the documentation string.